### ISSUE TITLE
Locative 2 (Bugzilla Bug 1704)

### ISSUE THREAD

**Issue by albbas** [giellatekno/oahpa#302]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 1704
Date: 2013-08-23T09:27:26+02:00
From: Tore Nesset &lt;tore.nesset&gt;
To:   Heli Uibo &lt;heli1401&gt;
CC:   @lajanda, tore.nesset, @Trondtr

Last updated: 2015-03-11T12:21:52+01:00

**Comment by albbas** [comment]
## Comment 8424
Date: 2013-08-23 09:27:26 +0200
From: Tore Nesset &lt;tore.nesset&gt;

How many words should we classify as having separate locative 2 forms in -у? The program gives forms like дню from день, which I believe is only found in the colloquial idiomatic expression на дню. I doubt that first year students would need to learn this (and, in fact, it may in fact damage their Russian, since it would encourage them to overuse the ending -у in syntactic constructions where it does not belong...)

In short, I think we need a more restrictive set of nouns with locative 2 endings.
